cmd/setup-gh.go

```diff
@@ -44,20 +39,6 @@ application and service principle, and will configure that application to trust
 
 			sc.AzClient.AzTenantClient = client
 
-			graphClient, err := createGraphClient(azCred)
```

Comment:
💡 Thanks for the ping, if there is any previous work which this work revert can you please att the commit hash for that here or in description of this PR, please :) thank you! will make it easy to quickly review. 

Thanks heaps again.

Reply:
I have added the reference PR's in the description above :)